### DISCUSSION TITLE
Blindlevel Up und Down mit true und false steuern

### DISCUSSION THREAD

**Discussion by tschortschde** [mcuiobroker/docuJarvis#34] [Q&A] [answered]
Hallo,
wie kann ein Datenpunktwert vom Blindlevel so umkonfiguriert werden, dass dieser bei "Hoch" den Wert "false" und bei "Runter" den Wert "true" an den iobroker übergibt?

**Comment by mcuiobroker** [comment, marked as answer]
Blindlevel gibt immer nur 100 oder 0 raus. In anderen Fällen muss ein Zusatzprogramm nutzen.
blindLevelUpDown, usw. siehe Doku.
Anscheinend doch mit "min" und "max".
![image](https://github.com/mcuiobroker/docuJarvis/assets/84818934/e8189ac7-4bb7-4f88-b3dc-1f61f53dd4d1)
`{"min":true,"max":false}`

**Reply by tschortschde** [reply]
Perfekt! ;-) 
Herzlichen Dank!
